docs/api/topics/apps.rst

```diff
@@ -115,13 +115,11 @@ App
             "regions": [
                 {
                     "adolescent": true,
-                    "mcc": 310,
```

Comment:
I guess we don't version our API docs? Would be nice to do that somehow. Maybe add something like `Changed in V2: "mcc" was removed from the output.` below this?